app/js/components/queryBuilder/queryParamSelector/queryParamSelector.component.js

```diff
@@ -159,27 +205,12 @@ const QueryParamSelectorComponent = {
         return vm.searchResults;
       };
 
-      vm.removeAllSearchResults = () => {
-        vm.searchResults = [];
-      };
-
-      /** @deprecated
-            // checks if set has been sorted by given params
-            // let isSorted = (list, sortedList, sortedOn)=>{
-            //     return _.isEqual(
-            //                 _.pluck(list, sortedOn),
-            //                 _.pluck(sortedList, sortedOn),
-            //             );
-            // };
-
-           
-            
-            /**
-             * @param  {Array} list- array of objects to sort  
-             * @param  {String} sortOn - object key to sort on 
-             * 
-             * @return {Array} 
-             */
+      /**
+       * @param  {Array} list- array of objects to sort
+       * @param  {String} sortOn - object key to sort on
+       *
+       * @return {Array}
+       */
       vm.sortResultsBy = (list, sortOn) => {
         vm.searchResults = sortedResultsBy(list, sortOn);
```

Comment:
Not really related to this PR: these two function names are good examples of bad naming: `sortResultsBy` and `sortedResultsBy`. Please feel free to add an issue to fix it later.

Reply:
Will do in the future 👍 These functions may need to be rewritten anywho.